LoopKit/QuantityFormatter.swift

```diff
@@ -173,22 +174,21 @@ public extension HKQuantity {
     }
 }
 
-public extension HKUnit {
+public extension LoopUnit {
     var usesMassFormatterForUnitString: Bool {
-        return self == .gram()
+        return self == .gram
     }
 
     var usesMeasurementFormatterForMeasurement: Bool {
-        return self == .gram()
+        return self == .gram
     }
 
     var preferredFractionDigits: Int {
         switch self {
         case .millimolesPerLiter,
-             HKUnit.millimolesPerLiter.unitDivided(by: .internationalUnit()),
-             HKUnit.millimolesPerLiter.unitDivided(by: .minute()):
-            return 1
-        case .milligramsPerDeciliterPerMinute:
```

Comment:
I don't believe this was working before... There's a test I'll highlight below that was passing while expecting this to be 0.